### PR TITLE
Beta 2.0

### LINKED ISSUE #3: Can't select new department
If you select department A, you cannot select another department until you deselect department A. Imo, it would be more intuitive if you could simply select the next department you wish to view, which may happen if a user is looking up multiple course syllabi.

selection_bug

### LINKED ISSUE #17: Group by course mnemonic
group individual classes by the course mnemonic (i.e MATH, CS, APMA, etc.)  instead of rendering each separately

### PR DESCRIPTION
**Pull request by verndrade**
- Fixed bug where new users would see a 500 Error because they violated a unique constraint on a blank field
- Added donate button
- Added department groups, split up by the course mnemonic
- Fixed school and index view showing a textbox rather than button
- Fixed bug where form would not refresh when non-pdfs were selected, allowing them to be uploaded
- Added custom admin site as Django admin site was broken
- Added privacy policy
- Added GitHub issue tracking for suggestions provided on feedback tab
- Added mass mail capabilities with security measures to ensure safe use

### CODE REVIEW

File: templates/school.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% extends 'base.html' %}
 {% block content %}
 
-{% if first %}
+{% if penis %}
```
Comment (winstliu) [inline templates/school.html]: :eyes:

**Review by winstliu — APPROVED** [review]
This is by-and-large a rubber stamp approval because I have no clue about the rest of the codebase.

What exactly is broken about the default admin panel?

File: templates/school.html (hunk 1 above)
Comment (verndrade) [reply templates/school.html]: lmao I have no idea what happened there. Don't leave your computer unlocked around children people and check your commits people

**Comment by verndrade** [comment]
> What exactly is broken about the default admin panel?

Basically I think postgres has a hard time with the table django uses for keeping track of sessions. It works locally when I use SQLite but when it is deployed the admin site requires me to login whenever I click a link and wont let me change any thing. Also I added some useful features to my admin site, like removing non .edu emails, recalculating the leaderboard, and mass email capabilities

Relevent StackOverflow link: https://stackoverflow.com/questions/32176587/django-admin-asks-for-login-after-every-click

**Pull request merged**